src/components/Tutorial/index.tsx

```diff
@@ -148,34 +102,9 @@ const Tutorial: FC = () => {
           <TutorialNavigation nextRef={nextRef} tutorialStep={tutorialStep} />
         </div>
 
-        {isTouch &&
-        (tutorialStep === TUTORIAL_STEP_FIRSTTHOUGHT ||
-          tutorialStep === TUTORIAL_STEP_SECONDTHOUGHT_HINT ||
-          tutorialStep === TUTORIAL_STEP_SUBTHOUGHT ||
-          tutorialStep === TUTORIAL2_STEP_CONTEXT_VIEW_TOGGLE ||
-          tutorialStep === TUTORIAL2_STEP_CONTEXT1_PARENT_HINT ||
-          (tutorialStep === TUTORIAL2_STEP_CONTEXT1_HINT &&
-            cursor &&
-            cursorHeadValue &&
-            cursorHeadValue.toLowerCase() === TUTORIAL_CONTEXT1_PARENT[tutorialChoice].toLowerCase()) ||
-          (tutorialStep === TUTORIAL2_STEP_CONTEXT1_SUBTHOUGHT_HINT &&
-            cursor &&
-            cursorHeadValue &&
-            cursorHeadValue.toLowerCase() === TUTORIAL_CONTEXT[tutorialChoice].toLowerCase()) ||
-          (tutorialStep === TUTORIAL2_STEP_CONTEXT2_PARENT_HINT &&
-            cursor &&
-            cursorHeadValue &&
-            cursorHeadValue.toLowerCase() === TUTORIAL_CONTEXT1_PARENT[tutorialChoice].toLowerCase()) ||
-          (tutorialStep === TUTORIAL2_STEP_CONTEXT2_HINT &&
-            cursor &&
-            cursorHeadValue &&
-            cursorHeadValue.toLowerCase() === TUTORIAL_CONTEXT2_PARENT[tutorialChoice].toLowerCase()) ||
-          (tutorialStep === TUTORIAL2_STEP_CONTEXT2_SUBTHOUGHT_HINT &&
-            cursor &&
-            cursorHeadValue &&
-            cursorHeadValue.toLowerCase() === TUTORIAL_CONTEXT[tutorialChoice].toLowerCase())) &&
-        gesturePath ? (
+        {isTouch && (
           <div
```

Comment:
The container needs to be hidden when the TutorialGestureDiagram is not rendered, otherwise a transparent black bar will appear. It may not be visible since it happens to be rendered in the empty space above the first thought, but it's probably better for it not to be rendered at all when there is no gesture being displayed.